calculation_scripts/OptimizationLoop.py

```diff
@@ -0,0 +1,167 @@
+import dda_model
+from dda_model import optimizers
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy.special import sici
+
+def _construct_test_model():
```

Comment:
FYI: The general rule in Python is that a leading underscore `_` indicates that the method is private to the module / test / script and not intended to be called outside that file. That's true here, so it's fine, just an FYI.